postView.html

```diff
@@ -1,12 +1,324 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/styles/index.css" />
+    <script src="https://use.fontawesome.com/350296bd22.js"></script>
+    <title>Home-Screen</title>
+  </head>
+
+  <body>
+    <header class="header--post-View px-1-5rem">
+      <div class="icon-container">
+        <svg
+          width="14"
+          height="12"
+          viewBox="0 0 14 12"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M5.66667 10.6666L1 5.99992M1 5.99992L5.66667 1.33325M1 5.99992H13"
+            stroke="white"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+        </svg>
+      </div>
+      <h1 class="text--uppercase">post</h1>
+    </header>
+
+    <main class="main">
+      <section class="post post--no-topline">
+        <div class="avatar">
+          <img
+            class="avatar__image"
+            src="https://source.unsplash.com/100x100/?photographer"
+            alt=""
+          />
+          <h2 class="avatar__heading">Tony Monta</h2>
+          <span class="avatar__timespan">1h ago</span>
+        </div>
+        <span class="post__burger-menu"></span>
+        <div class="post__text">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quae
+            possimus aut sed accusamus est obcaecati non vel dolore,
+            reprehenderit dolor
+          </p>
+        </div>
+        <div class="post__icons">
+          <span class="post__icon--thumb mr-1rem">2245</span>
+          <span class="post__icon--bubble mr-1rem">45</span>
+          <span class="post__icon--share mr-1rem">124</span>
+        </div>
+        <svg
+          aria-hidden="true"
+          focusable="false"
+          data-prefix="far"
+          data-icon="bookmark"
+          class="post__bookmark svg-inline--fa fa-bookmark"
+          role="img"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 384 512"
+        >
+          <path
+            fill="currentColor"
+            d="M336 0h-288C21.49 0 0 21.49 0 48v431.9c0 24.7 26.79 40.08 48.12 27.64L192 423.6l143.9 83.93C357.2 519.1 384 504.6 384 479.9V48C384 21.49 362.5 0 336 0zM336 452L192 368l-144 84V54C48 50.63 50.63 48 53.1 48h276C333.4 48 336 50.63 336 54V452z"
+          ></path>
+        </svg>
+      </section>
+
+      <section class="">
```

Comment:
Please look out for empty class attributes and delete them before making a PullRequest.